go-controller/pkg/node/management-port-dpu.go

```diff
@@ -47,15 +40,10 @@ func (mp *managementPortRepresentor) Create(nodeAnnotator kube.Annotator, waiter
 	// Get management port representor netdevice
```

Comment:
1 line above this, we can improve the print message by providing `mp.repName`. This would be good for debugging.